recipes/pysqlite3/run_test.py

```diff
@@ -0,0 +1,7 @@
+import pysqlite3.dbapi2 as sqlite3
```

Comment:
Is this file actually used?

Reply:
Yeah, it should be ran automatically (the same way `build.sh` is for the build step).

Reply:
Sorry - I did not knew this.